core/src/actors/json_rpc/json_rpc_methods.rs

```diff
@@ -144,6 +147,63 @@ pub fn get_block_chain() -> impl Future<Item = Value, Error = jsonrpc_core::Erro
         })
 }
 
+/// get output
+pub fn get_output(output_pointer: Result<(String,), jsonrpc_core::Error>) -> JsonRpcResultAsync {
```

Comment:
Why does this function operate over a result? Is it a callback or mapping function?